### PR TITLE
Parse GH Enterprise url only once

### PR DESCRIPTION
**Pull request by bfred-it**

### CODE REVIEW

File: background.js [hunk 1]
```diff
@@ -1,21 +1,13 @@
...
-    gh_enterprise = gh_enterprise.replace('https://', '');
-    gh_enterprise = gh_enterprise.replace('http://', '');
-    urls.push(gh_enterprise);
-  }
-
-  if (-1 !== urls.indexOf(host(tab.url))) {
+  if (urls.includes(host(tab.url))) {
     chrome.pageAction.show(tabId);
-    if (tab.url.indexOf('github.com') < 0) {
+    if (!tab.url.includes('github.com')) {
```
Comment (Yatser) [inline background.js]: Cool 👍

**Comment by Yatser** [comment]
We are all so used to having to write code that is usually supporting older/other browsers that we tend to forget that chrome extensions only have to work in, well, obviously chrome haha.

**Comment by Yatser** [comment]
Nice changes @bfred-it, I'll try to roll this up into a release soon

**Comment by bfred-it** [comment]
Right? I feel working on Chrome extensions rather liberating.

**Pull request merged**